### PR TITLE
Update loop unroll readme with hangcheck information

### PR DESCRIPTION
**Pull request by jkinsky**
# Existing Sample Changes
## Description

Updated readme with note on how to disable hangcheck. Added additional details about building on DevCloud.

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] VSCode

### CODE REVIEW

**Comment by jkinsky** [comment]
@pengtu - please review. (I cannot add you directly as a reviewer.)
@ishaghosh27 - please approve once Peng has approved.

File: DirectProgramming/DPC++/ParallelPatterns/loop-unroll/README.md
Comment (pengtu) [inline DirectProgramming/DPC++/ParallelPatterns/loop-unroll/README.md:113]: Looks good and thank you!

**Review by pengtu — APPROVED** [review]

**Review by ishaghosh27 — APPROVED** [review]

**Pull request merged**
